effekt/jvm/src/main/scala/effekt/EffektConfig.scala

```diff
@@ -6,6 +6,8 @@ import kiama.util.REPLConfig
 import org.rogach.scallop.{ ScallopOption, fileConverter, fileListConverter, longConverter, stringConverter, stringListConverter }
 
 class EffektConfig(args: Seq[String]) extends REPLConfig(args.takeWhile(_ != "--")) {
+  // Print version on `--version`
+  version(effekt.util.Version.effektVersion)
```

Comment:
If we prefix it with `"Effekt "`, then it wouldn't look so ugly in the help dialog and remains somewhat easy to parse.